src/ripple/basics/BasicConfig.h

```diff
@@ -76,6 +76,34 @@ class Section
         return values_;
     }
 
+    /**
+     * Set the legacy value for this section.
+     */ 
+    void
+    legacy(std::string value)
+    {
+        if (lines_.empty())
+            lines_.emplace_back(value);
+        lines_[0] = std::move(value);
```

Comment:
You're right, I forgot the move. You mean: `lines_.emplace_back(std::move(value));`.